### ISSUE TITLE
Update the ND image to 9.0.5.8

### ISSUE THREAD

**Issue by git4rk** [WASdev/azure.websphere-traditional.cluster#65]
tWAS 9.0.5.8 GAed and we should update the ND image to that level.

**Comment by edburns** [comment]
Hello @git4rk , consider [virtualimage.properties](https://github.com/WASdev/azure.websphere-traditional.image/blob/de79c0afb725cdd760f21b587e4609bf28e88050/twas-nd/src/main/scripts/virtualimage.properties#L14).  When we run the pipeline that runs the script which imports these properties, IBM Installation Manager will get the latest available version (9.0.5.8) at that point in time.  Also, when the cluster ARM template runs, [was-check.sh](https://github.com/WASdev/azure.websphere-traditional.image/blob/de79c0afb725cdd760f21b587e4609bf28e88050/twas-nd/src/main/scripts/was-check.sh#L46) is run, which includes this line:

```bash
output=$(${IM_INSTALL_DIRECTORY}/eclipse/tools/imcl updateAll -repositories "$REPOSITORY_URL" \
        -acceptLicense -log log_file -installFixes none -secureStorageFile storage_file -preferences $SSL_PREF,$DOWNLOAD_PREF -showProgress)
```

So I believe this already causes the 9.0.5.8 to be in the VHD files, we just need to update the Partner Center offer with the latest SAS urls from a pipeline run as recent as [this one](https://github.com/WASdev/azure.websphere-traditional.image/actions/runs/961493673) or newer.  

I assert this issue can be closed.

**Comment by majguo** [comment]
@git4rk The latest version of twas-nd base image 9.0.20210630 has already contains tWAS 9.0.5.8 and is referenced in the cluster offer. Pls verify and close the issue.
